code/datums/setup_option/backgrounds/origin/origin.dm

```diff
@@ -0,0 +1,101 @@
+/datum/category_group/setup_option_category/background/origin
+	name = "Origin"
+	category_item_type = /datum/category_item/setup_option/background/origin
+
+/datum/category_item/setup_option/background/origin
+
+/*
+	Shrimple guide to how this somehow operates, races outside of FBP and exalts no longer recieve race bonuses hidden away
+	but now within origins so that average player can view them without codediving.
+	Now how this is actually calculated, for each race bit differently.
+	Humans (regular) = 20 free points 0 minus points, doesn't apply to frontiersman  who uses alien calculation and vatgrown that has some minus points
+	Aliens (partially applies to naramadi) = 25 points 25 minus points, some exceptions might apply like mentioned naramadi but generally should be this
+	Why do humans recieve more points technically, simple, humans always were and will be generalists and with their weaker racials need something at least.
+	Aliens can easily afford to lose 25 points for 25 since they get insane starting bonuses compared to humans
+	Akula and Naramadi recieve 2 anatomy instead of their 10 toughness since they are supposed to be omegachonkers
+*/
+
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+/*
+		STAT_BIO = 2,
+		STAT_COG = 2,
+		STAT_MEC = 2,
+		STAT_ROB = 2,
+		STAT_TGH = 2,
+		STAT_VIG = 2
+*/
+
+/datum/category_item/setup_option/background/origin/spacer
+	name = "Spacer"
+	desc = "You come from a long line of spacefarers, explorers and general vagabounds not living on planets but space stations and ships their entire lives. \
+	Whichever might be your case, time spent in zero-g around your tightly knit community or perhaps entirely alone made you accustomed to it. \
+	You handle stresses of space way better than any surfacer and so does your rest of your body making you just tad bit safer in vastness of space."
+
+	perks = list(/datum/perk/lungs_of_iron)
+
+	restricted_to_species = list(FORM_HUMAN)
+
+	stat_modifiers = list(
+		STAT_ROB = 2,
+		STAT_TGH = 8,
+		STAT_VIG = 9,
+		STAT_BIO = 2,
+		STAT_MEC = 9,
+		STAT_COG = 2
+	)
+
+/datum/category_item/setup_option/background/origin/sol
+	name = "Sol-Born"
+	desc = "A catch all term to describe Humans and Abhumans born within inner borders of Sol, among alien races more of a deragotary slur, as such it is within certain circles \
+	viewed as a privilage. Most citizens recieve at least basic education compared to rest of the vast expanses of space so that they can become functioning members of society. \
+	However this pampered upbringing tends to hamper their chances of survival outside of Mother Sol."
+
+	perks = list(/datum/perk/solborn)
+
+	restricted_to_species = list(FORM_HUMAN, FORM_EXALT_HUMAN)
+
+	stat_modifiers = list(
+		STAT_ROB = 2,
+		STAT_TGH = 2,
+		STAT_VIG = 2,
+		STAT_BIO = 8,
+		STAT_MEC = 5,
+		STAT_COG = 13
```

Comment:
Why so much COG. If humans are generalists, I suggest spreading their stats across the board. I'd understand this much cognition out of a Mar'qua fleet born character